README.md

```diff
@@ -103,28 +104,29 @@ Supported:
 (e.g. Two overlapping webhook calls arrive and the flow has components where parallel processing > 1.)
 
 Limitations:
-* All groups must have one or more messages. (i.e. No groups of size 0). 
-Can't do re-grouping when a split is done on an empty array. (i.e. No empty for each pattern supported)
-* If all messages in a group fail to arrive at the re-assemble action (because one message suffered an error earlier in the flow)
-then this component will silently discard the group.
-* All messages must arrive within the same container lifetime.  
-If at any point there is more than a 15 second gap in messages, then the group will be silently discarded.  
+* All groups must have one or more messages. (i.e. No groups of size 0).
+Can't do re-grouping when a split is done on an empty array. (i.e. No empty for each pattern supported).
+* All messages must arrive within the same container lifetime.
```

Comment:
This limitation can now be removed (based on previous work).

Reply:
Are we talking about the group Id as message hier OR messageData?
In case it's about the group id, this limitations are still valid, if not then it can be removed.

Reply:
Done